src/coreclr/tests/tests.targets

```diff
@@ -54,7 +54,7 @@
 
       <CorerunExecutable Condition="'$(RunningOnUnix)' == 'true'"  >$(CORE_ROOT)\corerun</CorerunExecutable>
       <CorerunExecutable Condition="'$(RunningOnUnix)' != 'true'">$(CORE_ROOT)\corerun.exe</CorerunExecutable>
-      <CorerunExecutable Condition="'$(TargetArchitecture)' == 'wasm'"> $(DotnetRoot)/dotnet</CorerunExecutable>
+      <CorerunExecutable Condition="'$(TargetArchitecture)' == 'wasm' Or '$(TargetOS)' == 'Android'"> $(DotnetRoot)/dotnet</CorerunExecutable>
```

Comment:
I noticed that `CorerunExecutable` is also in `src/tests/run.proj`, is this line not needed there?

Reply:
That is for a different target `MonoAotCompileTests`, which is irrelevant here.